src/server.rs

```diff
@@ -1764,9 +1764,9 @@ impl Drop for Info {
 impl WaitUntilZero {
     #[rustfmt::skip]
     fn new() -> (WaitUntilZero, ActiveInfo) {
-        let info = Arc::new(Mutex::new(Info { waker: None }));
+        let _info = Arc::new(Mutex::new(Info { waker: None }));
```

Comment:
I don't get the underscore here

Reply:
This was just a convenience to keep using the shorthand struct init syntax; I'll revert that and use `let info` instead.